src/templates/base/2019/base_ebook.html

```diff
@@ -27,125 +32,101 @@
-        </section>
-      </section>
-
-      <section id="foreword">
-        <h1>{{ self.foreword_title() }}</h1>
-        {{ self.foreword() | replace('/%s/%s/contributors#' % (lang, year), '#contributors-') }}
-      </section>
-
-    {{ self.chapters() }}
-
-    <section class="chapter" id="methodology">
-      <div class="subtitle">Appendix A</div>
-      {{ get_ebook_methodology(lang,year) | safe }}
-    </section>
-
-    <section class="chapter" id="contributors">
-      <div class="subtitle">Appendix B</div>
-    
-        <h1 class="title title-lg">
-          {{ self.contributors_title() }}
-        </h1>
-        <div class="team">
-            <img src="/static/images/avatars/1.jpg" alt="" height="100" width="100" loading="lazy" />
-            <img src="/static/images/avatars/8.jpg" alt="" height="100" width="100" loading="lazy" />
-            <img src="/static/images/avatars/4.jpg" alt="" height="100" width="100" loading="lazy" />
-            <img src="/static/images/avatars/0.jpg" alt="" height="100" width="100" loading="lazy" />
-            <img src="/static/images/avatars/11.jpg" alt="" height="100" width="100" loading="lazy" />
-            <img src="/static/images/avatars/9.jpg" alt="" height="100" width="100" loading="lazy" />
         </div>
-        <p>{{ self.contributors_description() }}</p>
-        <ul class="contributors">
-        {% for id, contributor in config.contributors.items() %}
-        <li class="contributor" id="contributors-{{ id }}">
-          <img class="contributor-avatar" src="{{contributor.avatar_url}}" height="100" width="100" alt="{{ contributor.name }} avatar" />
-          <div>
+      </div>
+    </div>
+  </section>
+
+  <section id="foreword">
+    <h1>{{ self.foreword_title() }}</h1>
+    {{ self.foreword() | replace('/%s/%s/contributors#' % (lang, year), '#contributors-') }}
+  </section>
+
+  {{ self.chapters() }}
+
+  <section class="chapter" id="methodology">
+    <div class="subtitle">Appendix A</div>
+    {{ get_ebook_methodology(lang,year) | safe }}
+  </section>
+
+  <section class="chapter" id="contributors">
+    <div class="subtitle">Appendix B</div>
+  
+    <h1 class="title title-lg">
+      {{ self.contributors_title() }}
+    </h1>
+    <div class="team">
+        <img src="/static/images/avatars/1.jpg" alt="" height="100" width="100" />
+        <img src="/static/images/avatars/8.jpg" alt="" height="100" width="100" />
+        <img src="/static/images/avatars/4.jpg" alt="" height="100" width="100" />
+        <img src="/static/images/avatars/0.jpg" alt="" height="100" width="100" />
+        <img src="/static/images/avatars/11.jpg" alt="" height="100" width="100" />
+        <img src="/static/images/avatars/9.jpg" alt="" height="100" width="100" />
+    </div>
+    <p>{{ self.contributors_description() }}</p>
+    <ul class="contributors">
+    {% for id, contributor in config.contributors.items() %}
+      <li class="contributor" id="contributors-{{ id }}">
```

Comment:
A couple of suggestions for this section:
- remove the bullet symbol next to each contributor
- if possible, try to fit two per row

![image](https://user-images.githubusercontent.com/1120896/82126606-c465d100-977b-11ea-9cb9-10cb3c357bc3.png)

Reply:
> if possible, try to fit two per row

You don't know how many hours I spent on this in Weasyprint! With this it's as simple as `columns: 2`

Both done.